packages/fuels-contract/src/contract.rs

```diff
@@ -98,6 +94,7 @@ impl Contract {
         call_parameters: &CallParameters,
         compute_calldata_offset: bool,
     ) -> Result<(Vec<u8>, Vec<u8>), Error> {
+        use fuel_gql_client::fuel_types;
```

Comment:
I'm not sure where we are using `fuel_types` below?

Reply:
There is a macro hygiene issue with script-with-data-offset. I have a draft pr in fuel vm up that will make this no longer required.